src/index.js

```diff
@@ -1,32 +1,42 @@
 import './style.css';
 
-const board = document.getElementById('board');
+const sendData = async (url = '', data = {}) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    redirect: 'follow',
+    referrerPolicy: 'no-referrer',
+    body: JSON.stringify(data),
+  });
+  return response.json();
+};
+
+const nameInput = document.getElementById('name');
+const scoreInput = document.getElementById('score');
+const submitButton = document.getElementById('submit');
 
-const myScores = [{
-  name: 'Sean',
-  score: 100,
-},
-{
-  name: 'Dave',
-  score: 100,
-},
-{
-  name: 'Tshephang',
-  score: 100,
-}];
+submitButton.addEventListener('click', async () => {
+  await sendData('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/', { user: nameInput.value, score: +scoreInput.value });
+  nameInput.value = '';
+  scoreInput.value = '';
+});
```

Comment:
- [ ] Nice work so far. However, I added my name and score in the leader board form and submitted it, then I waited for the API to load and I searched for my record but couldn't find it. I also clicked on the refresh button and still my record was not found. I suggest you generate a new unique identifier for the game because the current one you have is filled up with hundreds of names that I have to use `ctrl f` to search for my record and still not found. Please try to fix this bug so that your app works as expected 😄.